### ISSUE TITLE
Uncolored prompt

### ISSUE THREAD

**Issue by davoaux** [NerdyPepper/pista#12]
I can't manage to get the prompt with colors. The binary shows up with colours though.

I'm using `zsh`

![20200706_190943](https://user-images.githubusercontent.com/37302363/86620530-c3ebf880-bfbc-11ea-9be6-0ef71d9a573d.png)

**Comment by oppiliappan** [comment]
Could you please share the relevant parts of your zsh config?

**Comment by davoaux** [comment]
Sure, [here](https://0x0.st/iydG.txt).

**Comment by oppiliappan** [comment]
Very strange, could you run zsh without any rcs:
```
zsh -d -f
```
and then set the prompt manually (just paste this at the shell) :
```
autoload -Uz add-zsh-hook
_pista_prompt() {
	PROMPT="$(pista -z)"   # `pista -zm` for the miminal variant
}
add-zsh-hook precmd _pista_prompt
```
and let me know if the issue persists? If colors show up, it probably means another program is messing with the colors.

I am not too familiar with zsh and recent updates to zsh, so this is a shot in the dark. Could you tell me the output of `$TERM` as well? I'm not too sure if that is relevant here, but it might be useful.

**Comment by davoaux** [comment]
Yes, the issue persists. It doesn't work using bash either. I tried diferent terminals (xterm and urxvt) and the result is the same.

![20200710_130854](https://user-images.githubusercontent.com/37302363/87149050-d497b980-c2af-11ea-9576-1dbf759a130b.png)

**Comment by oppiliappan** [comment]
> It doesn't work using bash either

Ah, It seems to be an issue with directly installing from crates.io, I am able to replicate it. For now, you can have colors work by installing from source:

```shell
git clone https://github.com/NerdyPepper/pista/ --recurse-submodules
cargo install --path ./pista --force
``` 
I would guess this has something to do with our use of submodules in the crate, but I am not sure how or why the colors vanish in a direct install, but not a manual install.

**Comment by Jimmysit0** [comment]
Same issue here

**Comment by bgrosse-midokura** [comment]
same with bash here, binary outputs color but in `PS1=$(pista)` it does not (crate install)
